pos-print/src/main/java/com/clouway/pos/print/adapter/db/PersistentCashRegisterRepository.kt

```diff
@@ -56,19 +56,31 @@ class PersistentCashRegisterRepository @Inject constructor(private val database:
   }
 
   private fun adapt(record: Document): CashRegister {
+    val policy = record["fiscalPolicy"] as Document
+    val policies = when {
+        policy.isEmpty() || policy == null -> mutableListOf(FiscalPolicy(0, 20.0))
```

Comment:
Will cause NPE if policy is null. 
policy == null || policy.isEmpty()

Reply:
fixed